pkg/assume/assume.go

```diff
@@ -115,6 +115,8 @@ func AssumeCommand(c *cli.Context) error {
 	activeRoleProfile := assumeFlags.String("active-aws-profile")
 	activeRoleFlag := assumeFlags.Bool("active-role")
 
+	clio.Info(activeRoleProfile, activeRoleFlag, os.Environ())
```

Comment:
Im guessing this was a debugging statement? Can be removed now

Reply:
Yes good catch, this can be removed